from requests.exceptions import HTTPError


class SummonerNotFoundError(Exception):
    pass
